addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -56,7 +56,8 @@ export class ChatWindow extends Component {
             !this.chatWindow.threadView ||
             !this.chatWindow.threadView.messageListView ||
             !this.chatWindow.threadView.messageListView.component ||
-            !this.chatWindow.threadViewer
+            !this.chatWindow.threadViewer ||
+            !this.chatWindow.threadView.messageListView.getScrollableElement()
```

Comment:
rearrange conditions to keep threadView and threadViewer together

Reply:
condition on `this.chatWindow.threadView.messageListView.component` probably becomes obsolete (included into the other one, component is never accessed explicitly below)